charts/wa-workflow-api/values.yaml

```diff
@@ -7,6 +7,8 @@ java:
     RESTART_APP: 1
     ENABLE_SWAGGER: true
     CAMUNDA_URL: "http://camunda-api-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal/engine-rest"
+    S2S_URL: "http://rpe-service-auth-provider-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal"
+    IDAM_URL: "https://idam-api.{{ .Values.global.environment }}.platform.hmcts.net"
```

Comment:
s2s and idam will probably follow a different naming convention in Prod. Therefore, these values need to be set in flux.  From what I remember, if you had a smoke test on Prod that talked to Idam it would probably fail because the idam URL is different. I am not sure about the s2s URL, though.

Reply:
Good spot David you're right I'll keep this in the todo bucket for when we have a prod env to make sure we don't miss these 🚀 👍